src/zzz_od/application/charge_plan/charge_plan_app.py

```
@@ -77,18 +89,18 @@ def find_and_select_next_plan(self) -> OperationRoundResult:
         如果找不到，返回计划完成状态。
         """
         # 检查是否所有计划都已完成
-        if self.ctx.charge_plan_config.all_plan_finished():
+        if self.config.all_plan_finished():
             # 如果开启了循环模式且所有计划已完成，重置计划并继续
```

Ruff (0.13.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)